### DISCUSSION TITLE
Feature checklist [Jul 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6043] [Discord Threads]
> 🙋‍♂️ **funapple**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Cell highlighting : No. We do have search & filter sans highlighting   
#### Drop down lists : Yes. Refer to datatype Single select / Multi Select & LinkToAnotherRecord (LTAR)   
#### True-false checkboxes : Yes. Refer to datatype checkbox  
#  
> 🙋‍♂️ **funapple**  
#### Cool enough. But if no highlighting, maybe there is some kind of automatic notifications is possible to create so it'll send notification to user/s if condition in cell is met?  
#  
> 👨‍🦲 **dstala**  
#### In-app user notifications are in our roadmap, not available yet.    
####    
#### Using webhook- Yes.    
#### https://docs.nocodb.com/developer-resources/webhooks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1115202674573447238)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
